docs/scu/dev/features/upgrade-verification.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Active detection of incompatibilies
+description: How Spryker Code Upgrader actively detects and warns you when your code becomes incompatible with the code in upgraded modules
+template: concept-topic-template
+---
+
+It is very important for Spryker Code Upgrader to ensure stability of the upgrades. On the one hand you have your automated tests that you rely on, those tests are run in your CI system and you use them before you merge the pull request created by Spryker Code Upgrader. On the other hand and in addition we run a number of code checks to offer an early warning system even without having a full context or knowledge of your project.
+
+# Validations and warnings
+
+The upgrader offers validations ensure the accuracy and safety of the upgrade process.
+The warnings provide guidance on potential risks or necessary actions for a successful upgrade.
+
+Warning types:
+- Major code releases warning.
+  All the major releases should be installed manually to integrate the BC breaks.
```

Comment:
Do we have instructions or does the warning include any instructions?

Reply:
Instructions are described below